backend/app/__init__.py

```diff
@@ -83,3 +83,88 @@ def one_item(item_id):
 @app.route("/")
 def index():
     return "Works :)", 200
+
+#DB for the people opinions/posts
+@app.route("/opinion", methods=["POST", "GET"])
+def opinions_list():
+    if request.method == "POST":
+        body = request.json
+        product_name = body["productName"]
```

Comment:
Product name is not actually needed, but maybe another endpoint for the search would help later if we are able to finish the MVP

Reply:
You are right, maybe I should delete that for now

Reply:
Yeah, maybe use product_id instead